lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -199,15 +198,23 @@ public void OnNext(IDriverStarted value)
         /// <param name="allocatedEvaluator">The allocated evaluator</param>
         public void OnNext(IAllocatedEvaluator allocatedEvaluator)
         {
-            Logger.Log(Level.Info, "AllocatedEvaluator EvaluatorBatchId [{0}], memory [{1}], systemState {2}.", allocatedEvaluator.EvaluatorBatchId, allocatedEvaluator.GetEvaluatorDescriptor().Memory, _systemState.CurrentState);
+            Logger.Log(Level.Info, "AllocatedEvaluator memory [{0}], systemState {1}.", allocatedEvaluator.GetEvaluatorDescriptor().Memory, _systemState.CurrentState);
             lock (_lock)
             {
                 using (Logger.LogFunction("IMRUDriver::IAllocatedEvaluator"))
                 {
                     switch (_systemState.CurrentState)
                     {
                         case SystemState.WaitingForEvaluator:
-                            _evaluatorManager.AddAllocatedEvaluator(allocatedEvaluator);
+                            if (!_evaluatorManager.IsMasterEvaluatorAllocated())
```

Comment:
We use it in positive form later (in log). I'd keep method name as is, it's harder to process method name with a verbal negation than to read !positive in code (at least for me).